modules/compliance-operator-configure-scanning.adoc

```diff
@@ -7,31 +7,23 @@
 = Configuring the ScanSettingBinding object
 
 [role="_abstract"]
-Create a `ScanSettingBinding` object in the `openshift-compliance` namespace to scan the cluster by using the `cis` and `cis-node` profiles.
+By creating a `ScanSettingBinding` object in the `openshift-compliance` namespace, you can scan your cluster by using the `cis` and `cis-node` profiles either from the command-line interface (CLI) or user interface (UI).
```

Comment:
I think we should separate CLI and UI more.  If you use the CLI the results will mostly be on the Dashboard for now.  If you use the UI results will appear in the tech preview coverage page as well as the dashboard.  We should also note that the UI is tech preview while the CLI is the currently fully supported means.